modules/user_manual/pages/session_management.adoc

```diff
@@ -1,36 +1,19 @@
-= Manage Connected Browsers and Devices
+= Session Management
 
-The personal settings page allows you to have an overview on the
-connected browsers and devices.
+The personal settings page allows you to have an overview of the connected browsers and clients.
 
-[[managing-connected-browsers]]
-== Managing Connected Browsers
+image:settings_sessions.png[Sessions]
 
-In the list of connected browsers you see which browsers connected to
-your account recently:
+[[sessions]]
+== Sessions
 
-image:settings_sessions.png[List of browser sessions.]
+In the list of sessions you see which browsers and clients are connected to your account recently. You can use the trash icon to terminate any of the sessions in the list. If you want to lock out a user, you would need to change his password and then terminate his session. You can not terminate the session you are using to access the sessions page.
```

Comment:
I propose to put the sentence: `You can not terminate the session...` into a note:
`NOTE: You can not terminate the session...`
"Note: ..." needs to be in a new line with a blank line in between the last sentence.

Reply:
I'd rewrite this as follows:

```asciidoc
The sessions list shows which browsers and clients were recently, and are actively connected to your ownCloud installation. You can use the trash icon, at the far right-hand side of any list entry, to terminate that session. 

[NOTE]
====
*Please be aware of two things:*

. If you want to lock out a user, you need to change their password *before* you terminate their session. 
. The user cannot terminate their active session from the sessions list.
====
```

Reply:
Sorry about the formatting, but I thought it easier to just dump the info as formatted AsciiDoc and save you some time.

Reply:
Is this an Enterprise-only feature? The trash icon's not available in the community version that I'm testing with.

Reply:
how many connection do you have to your ownCloud?

Reply:
Can you clarify what you mean? I don't understand what you're asking.

Reply:
> Is this an Enterprise-only feature? The trash icon's not available in the community version that I'm testing with. If so, this section should be updated to make that clear.

No, it's not an enterprise feature. But if you have only one connection to your ownCloud - if you setup an oC and connect to it with Chrome = that's one session. That one you can't terminate = you don't see the trash icon.
If you now connect to your oC with your iPhone/Sync Client, you can terminate those sessions = the trash icon appears.

Reply:
That wasn't the case for me the last time I tried it. I'll check again to see if I just overlooked something.

Reply:
Okay, perhaps I was not thinking properly when I wrote that comment (likely nervous before going to the dentist), but it's working, as stated, now. Thanks for calling me out on that, @voroyam.